test/ZNSCurvePricer.test.ts

```diff
@@ -50,14 +51,22 @@ describe("ZNSCurvePricer", () => {
       randomAcc,
     ] = await hre.ethers.getSigners();
 
+    const priceConfig : ICurvePriceConfig  = DEFAULT_PRICE_CONFIG;
```

Comment:
Small nitpick, doesn't make a huge difference but assigning this constant to a variable and then using it only one time is unnecessary.  Just give `DEFAULT_PRICE_CONFIG` itself to the `deployZNS` command below